release_notes/addtl-release-notes.adoc

```diff
@@ -15,15 +15,13 @@ Release notes for additional related components and products not included in the
 The following release notes are for downstream Red Hat products only; upstream or community release notes for related products are not included.
 ====
 
-[cols="1a,1a,1a",frame=none,grid=none]
-|===
-|A::
+A::
 xref:../networking/aws_load_balancer_operator/aws-load-balancer-operator-release-notes.adoc#aws-load-balancer-operator-release-notes[AWS Load Balancer Operator]
 
-|B::
+B::
```

Comment:
@adellape and @bergerhoffer 

- Should this not be under O, as it is part of OpenShift ?
- and the name should start with a small case and not capital B, it must be `builds for Red Hat OpenShift`, correct?

Reply:
I was just going by the common-attribute `{builds-v2title}`, which comes out as "Builds for Red Hat OpenShift".

And it's under "B" on https://docs.redhat.com/en/products.

Reply:
@Srivaralakshmi Yeah the other OCP operators aren't under "O" because they're all OpenShift, they're under "O" because it's the next letter alphabetically once you take out "Red Hat" from consideration alphabetically.

For example, the "Secondary Scheduler Operator for Red Hat OpenShift" is also an OpenShift operator, but it's under "S".